library/src/main/java/com/chuckerteam/chucker/internal/support/DateJsonAdapter.kt

```diff
@@ -0,0 +1,22 @@
+package com.chuckerteam.chucker.internal.support
+
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import java.text.DateFormat
+import java.util.Date
+import java.util.Locale
+
+class DateJsonAdapter {
+
+    @ToJson
+    fun toJson(date: Date): String {
+        val dateFormat = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, Locale.US)
```

Comment:
Why don't you create one variable for date format instead of copying it multiple times?

Reply:
Fixed.